File: 00_Data_parsing_server_part_II/01_node/Person.js

class Person {
    constructor(nameInput, ageInput, hobbiesInput) {
        this.name = nameInput;
        this.age = ageInput;
        this.hobbies = hobbiesInput;
    }

    displayPerson() {
        console.log(`name: ${this.name} , age: ${this.age} , hobbies: ${this.hobbies.toString()}`);
    }
}

export default Person;

File: 03a_Individual_data_parsing_server/serverB/Person.cs
public class Person
{
    public string Name { get; set; }
    public int Age { get; set; }

    public string[] Hobbies { get; set; }

    public Person(string name, int age, string[] hobbies)
    {
        Name = name;
        Age = age;
        Hobbies = hobbies;
    }

    public Person()
    {
        
    }
}